keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -422,20 +431,43 @@ type ClusterNetworkPolicyIngress/EgressRule struct {
 	Name         string
 }
 
-type ClusterNetworkPolicyType string
+// WorkloadSelector selects workloads based on the label Selector.
+// Right now the only selector type supported is PodSelector. More selector types may be
+// added in the future to support other workload selection mechanism, e.g. serviceAccountSelector.
+// Must be accompanied by a NamespaceSelector to specify the scope of the workload selection.
+type WorkloadSelector struct {
+	PodSelector         *metav1.LabelSelector
+}
+```
 
-// Add future selector types here to ease introduction of future selectors to ensure api fails 
-// closed cleanly if a CNI doesnt support specific types of selectors
+**Alternatives considered for the WorkloadSelector struct:**  
```

Comment:
if we don't have a major preference of this alternate .. we should simply mention this alternate here and put the details in a separate section within the alternatives considered section and add a reference here.. this would make the design review a bit more cleaner exp